makeWindowsRelease.py

```diff
@@ -26,7 +26,7 @@
 gTemporaryBuildDir = r'C:\temp_release'
 
 # External programs
-inkscapeDirList = [r'C:\Program Files\Inkscape', r'C:\Program Files (x86)\Inkscape']
+inkscapeDirList = [r'C:\Program Files\Inkscape\bin', r'C:\Program Files\Inkscape', r'C:\Program Files (x86)\Inkscape']
```

Comment:
Should we also add a bin path to the 32-bit version?

Reply:
Yes I will add that